system/Language/en/Validation.php

```diff
@@ -53,6 +53,7 @@
     'required'              => 'The {field} field is required.',
     'required_with'         => 'The {field} field is required when {param} is present.',
     'required_without'      => 'The {field} field is required when {param} is not present.',
+    'required_with_value'   => 'The {field} field is required when field and value has {param} is present.',
```

Comment:
This message is too bad. Please change to one that is easy to understand for humans.

> The identity_number field is required when field and value has is_internal,1 is present.

Reply:
how about this one
`'required_if' => 'The {field} field is required when the field specified in {param} is present.',`
> The identity_number field is required when the field specified in is_internal,1 is present. 

or 
`'required_if' => 'The {field} field is required when the {param} field has the specified value.',`
> The identity_number field is required when the is_internal,1 field has the specified value.

Reply:
or this one:

'required_if' => 'The {field} field is required when the {param} field has the given value.',
> The identity_number field is required when the is_internal,1 field has the given value.

Reply:
I think this is good.

> The identity_number field is required when the is_internal field is the given value.